modules/quant_api/src/EventSubscriber/QuantApi.php

```diff
@@ -177,14 +178,40 @@ public function onOutput(QuantEvent $event) {
         $fileOnDisk = str_replace('/system/files', $privatePath, $file);
       }
 
+      // Check if the file has changed.
       if (isset($item['existing_md5'])) {
         if (file_exists($fileOnDisk) && md5_file($fileOnDisk) == $item['existing_md5']) {
           continue;
         }
       }
 
-      // If the file exists we send it directly to quant otherwise we add it
-      // to the queue to generate assets on the next run.
+      // In Drupal 10.1, work around CSS/JS aggregation issues.
+      // Only process internal CSS/JS files.
+      $check_file = preg_match('/\.(css|js)$/', $file) && (strpos($item['original_path'], "http") !== 0);
```

Comment:
```suggestion
      $check_file = preg_match('/\.(css|js)$/', $file) && str_starts_with($item['original_path'], "http");
```
PHP8 introduces [str_starts_with](https://www.php.net/manual/en/function.str-starts-with.php) for readability we should look to use it. It's been polyfilled back to older versions as well so should be pretty safe to use.

https://core.trac.wordpress.org/ticket/58012

Reply:
It also introduced [str_ends_with](https://www.php.net/manual/en/function.str-ends-with.php) — this doesn't support multiple comparisons but would also be easier than the regex to maintain and using native language constructs may perform better.

```suggestion
      $check_file = (str_ends_with($file, 'css') || str_ends_with($file, 'js')) && (strpos($item['original_path'], "http") !== 0);
```

Reply:
Ah, I saw that function but didn’t realize it was back filled so I was worried about using it. I’ll get this in and merged and create a new release with the 2 D10 fixes 👍